src/windows/top_toolbar.c

```diff
@@ -1461,10 +1462,11 @@ static void window_top_toolbar_scenery_tool_down(short x, short y, rct_window *w
 	case SCENERY_TYPE_SMALL:
 	{
 		int quantity = 1;
-		if (window_scenery_is_build_cluster_tool_on) {
-			quantity = 35;
+		bool isCluster = window_scenery_is_build_cluster_tool_on && network_can_perform_action(network_get_current_player_group_index(), GAME_COMMAND_TOGGLE_SCENERY_CLUSTER);
+		if (isCluster) {
+			quantity = 36;
```

Comment:
That was by accident, i thought i read it as 35.  Like, i have a forked version of the code that contains a bunch of experimental features (autosave enable/disable checkbox in file menu, private messaging, customizable kicking messages, object selection modified to work in multiplayer, logging players IP and name when they join, a layout designer for designing and layout out widgets for windows) that contains a "_project_flags.h" file with a bunch of defines to enable/disable/isolate features i add in. When I submitted this code I created the wolfreak100 account to have 2 forks of the project, that way i could implement this feature only and send it, so when i copied the parts over i must have overlooked 35 as 36.

I've been meaning to go back and edit it to revise it, but lately i noticed the slowdown of accepting of pull requests so i've not been in really much of a rush tbh.